include/lsst/afw/geom/SphPoint.h

```diff
@@ -0,0 +1,319 @@
+// -*- LSST-C++ -*-
+
+/*
+ * LSST Data Management System
+ * See COPYRIGHT file at the top of the source tree.
```

Comment:
The current documentation guidelines require it: https://developer.lsst.io/docs/cpp_docs.html#boilerplate-for-header-h-and-source-cpp-files